count/Feature/ListeningQuizNavigationFeature.swift

```diff
@@ -0,0 +1,127 @@
+import ComposableArchitecture
+import SwiftUI
+
+struct ListeningQuizNavigationFeature: Reducer {
+    struct State: Equatable {
+        var listeningQuiz: ListeningQuizFeature.State
+        var path = StackState<Path.State>()
+        @PresentationState var settings: SettingsFeature.State?
```

Comment:
I'm not planning on adding any other navigation destinations at the moment, so I refactored this to be a single piece of state instead of a full `Destination` reducer.